app/templates/404.html

```diff
@@ -3,5 +3,38 @@
 
 {% block error_content %}
     <h1>Error 404</h1>
-    <h3>The page you're looking for was not found. Please head back <a href="/">home</a>.</h3>
+    <h3>The page you're looking for was not found. You can head back <a href="/">home</a> or you can stay on this page and find then click the crazy box below.</h3>
+    <hr style="width:90%;" align="left">
+    <div id="container" style="height: 450px;">
+    	<div id="noli_me_tangere" class="crazy-box" onClick="show_popup()" onmouseover="randommove()">You didn't found me, did ya?</div>
```

Comment:
didn't *find* — you decline the first  verb already; the second stays in base form